# muscle state
string name
float32 force
float32 length
float32 lengthening_speed
geometry_msgs/Vector3[] path_points
